source/localise-govuk-frontend/index.html.md.erb

```diff
@@ -0,0 +1,137 @@
+Different languages have different rules for pluralisation.
+
+English only has 2 forms, singular and plural. Other languages often have more forms. For example, Welsh has 6 possible plural forms, the Romance languages (French, Spanish, Italian) have 3 and many East Asian languages (Chinese, Japanese, Malaysian) do not have any plural forms.
+
+We follow the [Unicode Common Locale Data Repository classification](https://cldr.unicode.org/index/cldr-spec/plural-rules) and categorise these forms as: `zero`, `one`, `two`, `few`, `many` and `other`.
+The forms for each language will differ depending on its pluralisation. For example,
+
+- all languages use the `other` form
+- some languages will require you to pass additional forms
+- languages with no plural form only need the form `other`
+
+You should use a professional translation service to make sure your translation is accurate and that you're using the correct pluralisation form. You can also check [the Unicode Common Locale Data Repository documentation for guidance on which of these forms your locale requires and what `few` and `many` might represent for it](https://unicode-org.github.io/cldr-staging/charts/latest/supplemental/language_plural_rules.html).
+
+When providing text with plural forms, you'll need to pass an object associating each plural category with the text for that form. If you do not provide the required plural form for the locale, the component will fall back to the “other” form and warn you in the browser's console.
+
+If you're looking to keep English messages and override only one of the plural forms (`one` or `other`), you can only provide that specific form and it will be merged with our defaults.
+
+Our components will replace the `%{count}` placeholder with the number used for picking the plural category.
+
+The following example shows how to provide pluralisation options using the different ways you can [configure components with JavaScript](/configure-components-with-javascript/):
+
+With Nunjucks
+
+```njk
+{{ govukCharacterCount({
+  charactersUnderLimitText: {
+    other: '%{count} characters to go'
+    one: 'One character to go'
+  },
+  charactersOverLimitText: {
+    one: 'One character too many'
+  }
+})
+{# The `other` key for `characterOverLimitText` will be the default one #}
+```
+
+With data attributes
+
+```html
+<div data-module=”character-count” 
+  data-i18n.characters-under-limit.other=”%{count} characters to go”
+  data-i18n.characters-under-limit.one=”One character to go”
+  data-i18n.characters-over-limit.one=”One character too many”>
+  <!-- The `other` key for `characterOverLimitText` will be the default one -->
+```
+
+With JavaScript
+
+```js
+new CharacterCount($element, {
+  i18n: {
+    charactersUnderLimit: {
+      other: '%{count} characters to go'
+      one: 'One character to go'
+    },
+    charactersOverLimitText: {
+      one: 'One character too many'
+      // The `other` key for `characterOverLimitText` will be the default one
```

Comment:
We have `other` as mandatory in our JSDoc so leave it out here

Reply:
The JSDoc had an error for that one as you may want to pass an object without the `other` key when overriding only one of our English defaults, like in the example.